src/controller/MazeController.java

```diff
@@ -0,0 +1,68 @@
+package controller;
+
+import model.MazeModel;
+import view.CLI;
+
+import java.io.IOException;
+
+public class MazeController {
+
+    private CLI cli_view;
+    private MazeModel maze_model;
+
+    public Observable VCObservable;
```

Comment:
```suggestion
    // Observer pattern control variables. VC stands for View-Controller (V-View, C-Controller, M-Model)
    public Observable VCObservable;
```

Reply:
Also add a word about Observer vs Observable

Reply:
Done

Reply:
Didn't see your exp about the observer patter itself

Reply:
V